src/phys/dca_step/cluster_solver/shared_tools/accumulation/tp/ndft/ndft_kernels.cu

```diff
@@ -136,14 +136,17 @@ void rearrangeOutput(const int nw, const int no, const int nb, const std::comple
 }
 
 // Explicit instantiation.
-template void sortM<double, double>(int, const double*, int, std::complex<double>*, const int,
+template void sortM<double, double>(int, const double*, int, std::complex<double>*, int,
                                     const Triple<double>*, const Triple<double>*, const cudaStream_t);
-template void sortM<float, float>(int, const float*, int, std::complex<float>*, const int,
-                                  const Triple<float>*, const Triple<float>*, const cudaStream_t);
+template void sortM<double, float>(int, const double*, int, std::complex<float>*, int,
```

Comment:
Where is the unit test coverage for this?

Reply:
This is an implementation detail of CachedNdft(Cpu/Gpu), In 204ba89b17519cc90008cd20b5a7624671cd4700 I test them both in single and double precision.